new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleTraefikValidation.java

```diff
@@ -0,0 +1,678 @@
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(domainExists(domainUid, DOMAIN_VERSION, domainNamespace));
+
+    // check admin server pod exist
+    logger.info("Check for admin server pod {0} existence in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodCreated(adminServerPodName);
+
+    // check managed server pods exists
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check for managed server pod {0} existence in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodCreated(managedServerPrefix + i);
+    }
+
+    // check admin server pod is running
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReady(adminServerPodName);
+
+    // check managed server pods are running
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodReady(managedServerPrefix + i);
+    }
+
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkServiceCreated(adminServerPodName);
+
+    // check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managed server service {0} is created in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkServiceCreated(managedServerPrefix + i);
+    }
+  }
+
+  @Test
+  @Order(2)
+  @DisplayName("Create an ingress per domain")
+  public void testCreateIngress() {
+    // Helm install parameters for ingress
+    ingressParam = new HelmParams()
+            .releaseName(domainUid + "-ingress")
+            .namespace(domainNamespace)
+            .chartDir(projectRoot + "/../kubernetes/samples/charts/ingress-per-domain");
+
+    assertThat(createIngress(ingressParam, domainUid, domainUid + ".org"))
+            .as("Test createIngress returns true")
+            .withFailMessage("createIngress() did not return true")
+            .isTrue();
+
+    // check the ingress is created
+    String ingressName = domainUid + "-traefik";
+    assertThat(assertDoesNotThrow(() -> getIngressList(domainNamespace)))
+        .as(String.format("get the ingress %1s in namespace %2s", ingressName, domainNamespace))
+        .withFailMessage(String.format("can not get ingress %1s in namespace %2s", ingressName, domainNamespace))
+        .contains(ingressName);
+
+    logger.info("ingress is created in namespace {0}", domainNamespace);
+  }
+
+  @Test
+  @Order(3)
+  @DisplayName("Verify the application can be accessed through the ingress controller ")
+  public void testSampleAppThroughIngressController() throws Exception {
+    String hostname = InetAddress.getLocalHost().getHostName();
```

Comment:
using local host will not work on multi node cluster or Jenkins. I gave this suggestion to Pani in his PR - 
Add this to TestConstants.java and use K8S_NODEPORT_HOST in place of host name.
```
 public static final String K8S_NODEPORT_HOST = Optional.ofNullable(System.getenv("K8S_NODEPORT_HOST"))
      .orElse(InetAddress.getLocalHost().getHostName());
```